br/pkg/restore/data.go

```diff
@@ -439,7 +536,7 @@ func (recovery *Recovery) makeRecoveryPlan() {
 			for i, m := range x {
 				log.Debug("make plan", zap.Uint64("storeid", m.storeId), zap.Uint64("regionid", m.RegionId))
 				plan := &recovpb.RecoverCmdRequest{RegionId: m.RegionId, AsLeader: (i == 0)}
-				// max last index as a leader
+				// sorted by log term -> last index -> commit index in a region
 				if plan.AsLeader {
 					log.Debug("as leader peer", zap.Uint64("storeid", m.storeId), zap.Uint64("regionid", m.RegionId))
 					recovery.recoveryPlan[m.storeId] = append(recovery.recoveryPlan[m.storeId], plan)
```

Comment:
please fix abbr in https://github.com/pingcap/tidb/blob/c6ca2589e9dfa5199f1cbf9829f525c6b87c70b3/br/pkg/restore/data.go#L421